integration-tests/cypress.config.js

```diff
@@ -5,13 +5,13 @@ module.exports = defineConfig({
   viewportHeight: 1080,
   screenshotsFolder: './screenshots',
   videosFolder: './videos',
-  video: false,
+  video: true,
```

Comment:
Changing `video:true` to help with debugging. Although Sam recommended in the past that we leave this off but I think it would be helpful for debugging the issue. Can remove later.